backend/controllers/github.go

```
@@ -1120,6 +974,158 @@ func handleIssueCommentEvent(gh utils.GithubClientProvider, payload *github.Issu
 	return nil
 }
 
+func GenerateTerraformFromCode(payload *github.IssueCommentEvent, commentReporterManager utils.CommentReporterManager, config *dg_configuration.DiggerConfig, defaultBranch string, ghService *dg_github.GithubService, repoOwner string, repoName string, commitSha *string, issueNumber int, branch *string) error {
+	if strings.HasPrefix(*payload.Comment.Body, "digger generate") {
+		projectName := ci.ParseProjectName(*payload.Comment.Body)
+		if projectName == "" {
+			commentReporterManager.UpdateComment(fmt.Sprintf(":x: generate requires argument -p <project_name>"))
+			log.Printf("missing project in command: %v", *payload.Comment.Body)
+			return fmt.Errorf("generate requires argument -p <project_name>: %v")
```

golangci-lint (1.62.2)

printf: fmt.Errorf format %v reads arg #1, but call has 0 args

(govet)